cmd/otelcol/config/collector/splunk_config_linux.yaml

```diff
@@ -77,7 +74,7 @@ service:
     traces:
       receivers: [jaeger, opencensus, otlp, sapm, zipkin]
       processors: [memory_limiter, batch]
-      exporters: [sapm, signalfx_correlation]
+      exporters: [sapm]
```

Comment:
Is this for the gateway case instead of agent? If so correlation shouldn't be enabled so this is fine.

Reply:
Not sure. I just assumed `signalfx_correlation` should be removed from the configs since the exporter was removed upstream.

Reply:
I think it's fine to keep it removed from this one.